### PR TITLE
chore: add claimed for 8, 9 round

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- adds an exception for round 8 & 9 to show as claimed in the history section
- ensures this doesn't appear in the ready to claim section

##### Refers/Fixes

closes https://github.com/gitcoinco/web/issues/10610

##### Testing

Locally tested

### CODE REVIEW

File: app/assets/v2/js/grants/ready_claim.js [hunk 1]
```diff
@@ -18,7 +18,8 @@ Vue.component('matching-claim', {
       return this.status === 'pending';
     },
     isKYCPending() {
-      return !this.match.ready_for_payout;
+      console.log(this.match.grant_payout.status == 'ready')
```
Comment (gravityblast) [inline app/assets/v2/js/grants/ready_claim.js:21]: I think we left a debug line

**Review by gravityblast — APPROVED** [review]

**Comment by gravityblast** [comment]
LGTM

**Comment by schultztimothy** [comment]
Just a couple linting errors but otherwise LGTM!

**Review by schultztimothy — APPROVED** [review]

File: app/assets/v2/js/grants/ready_claim.js (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/v2/js/grants/ready_claim.js:21]:
```suggestion
```

**Comment by thelostone-mc** [comment]
thanks guys <3

File: app/assets/v2/js/grants/ready_claim.js [hunk 2]
```diff
@@ -18,7 +18,8 @@ Vue.component('matching-claim', {
       return this.status === 'pending';
     },
     isKYCPending() {
-      return !this.match.ready_for_payout;
+      console.log(this.match.grant_payout.status == 'ready');
```
Comment (thelostone-mc) [inline app/assets/v2/js/grants/ready_claim.js:21]:
```suggestion
```

**Pull request merged**